src/Workspaces/CoreTest/CodeCleanup/NormalizeModifiersOrOperatorsTests.cs

```diff
@@ -721,7 +721,7 @@ Private _
             var expected = @"Class A
     Private _
         Shared _
- _
+               _
```

Comment:
😕 It seems like this one should have been removed altogether

```suggestion
```

Reply:
@sharwell They were not in the original design even before I added "comments" after Line Continuation all I did was move the underscore to a more appropriate place. Many of the tests expect the same results with out without the comments. You can see the original behavior on line 724 before my change.
```
        <Theory, Trait(Traits.Feature, Traits.Features.Formatting)>
        <InlineData("_")>
        <InlineData("_ ' Comment")>
```

Reply:
Makes sense. We should file a follow-up issue to prevent the extraneous continuation as a separate change.

Reply:
@sharwell  Agreed, I think it is rare someone would actually do it without a comment but I can enter an issue. It will require duplicating all the tests which today use InlineData.

Reply:
@sharwell This might be better handled by IDE2000 and/or RS0101 and the code fixes for them as a separate PR. Today I can type lots of blank lines and nothing stops me but I do get the two warmings above.